application_endpoints.md

```diff
@@ -194,3 +197,39 @@ Key | Type | Description
 ----|------|------------
 `username` | `string` | The unique username of the user.
 `status` | `string` | The user's status.
+
+#### Resource Events (Contact, etc)
+
+Payload:
+
+Key | Type | Description
+----|------|------------
+`resource_type` | `string` | "Contact", etc
+`resource_id` | `string` | The id of the resource that caused the event.
+`event` | `string` | The short form of the resource event.
+`username` | `string` | The unique username of the user. **Will be deprecated.**
+`user_id` | `string` | The unique id of the user. **Will be deprecated.**
+`account_id` | `string` | The unique id of the account.
+
+An example contact creation:
+
+```
+POST /payload HTTP/1.1
+
+Host: yourserver.com
+User-Agent: Outstand-Webhook
+Content-Type: application/json
+Content-Length: <length>
+X-Outstand-Event: create_contact
+X-Outstand-Id: 123
+X-Outstand-Signature: sha256=<signature>
+
+{
+  "resource_type": "Contact",
+  "resource_id": "21",
+  "event": "create",
+  "username": "user12345",
+  "user_id": "4862",
```

Comment:
I went with strings for all of the ids here but I don't think the code we just shipped does this. Thoughts?

Reply:
We're going with all string ids.